docs/modules/ROOT/pages/developers.adoc

```diff
@@ -63,7 +63,7 @@ To build the whole project you now need to run:
 make
 ----
 
-This execute a full build of both the Java and Go code. If you need to build the components separately you can execute:
+This executes a full build of both the Java and Go code. If you need to build the components separately you can execute:
```

Comment:
I thin we can remove the reference to Java

Reply:
Right. Updated.